servicetalk-http-api/src/main/java/io/servicetalk/http/api/StreamingHttpResponse.java

```diff
@@ -46,11 +46,24 @@
 
     /**
      * Gets a {@link Publisher} that combines the raw payload body concatenated with the {@link HttpHeaders trailers}.
+     * @deprecated Use {@link #messageBody()}.
      * @return a {@link Publisher} that combines the raw payload body concatenated with the
      * {@link HttpHeaders trailers}.
      */
+    @Deprecated
     Publisher<Object> payloadBodyAndTrailers();
```

Comment:
This method is still used in `StreamingHttpServiceToBlockingStreamingHttpService#handleBlockingRequest`, please update it to use `messageBody()`